server/src/service.rs

```diff
@@ -83,9 +83,32 @@ impl Service {
     #[zbus(out_args("unlocked", "locked"))]
     pub async fn search_items(
         &self,
-        _attributes: HashMap<&str, &str>,
+        attributes: HashMap<String, String>,
     ) -> Result<(Vec<OwnedObjectPath>, Vec<OwnedObjectPath>), ServiceError> {
-        todo!()
+        let mut unlocked = Vec::new();
+        let mut locked = Vec::new();
+        let collections = self.collections.lock().await;
+
+        for collection in collections.iter() {
+            for item in collection.search_inner_items(&attributes).await {
+                if item.is_locked().await {
+                    locked.push(item.path().clone());
+                } else {
+                    unlocked.push(item.path().clone());
+                }
+            }
+        }
+
+        if unlocked.is_empty() && locked.is_empty() {
+            tracing::info!(
```

Comment:
Both those tracings should be debug imho

Reply:
same for the ones you added in collection.rs

Reply:
Done, thanks!